user-restricted-separate-auth-tutorials/cis2/csharp/HelloWorldSeparateAuth/Pages/Error.cshtml.cs

```diff
@@ -0,0 +1,27 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.Diagnostics;
+
+namespace HelloWorldCombinedAuth.Pages
+{
+    [ResponseCache(Duration = 0, Location = ResponseCacheLocation.None, NoStore = true)]
+    [IgnoreAntiforgeryToken]
+    public class ErrorModel : PageModel
+    {
+        public string? RequestId { get; set; }
+
+        public bool ShowRequestId => !string.IsNullOrEmpty(RequestId);
+
+        private readonly ILogger<ErrorModel> _logger;
```

Comment:
I noticed pretty all `_logger` fields are unused. I'm not sure if it's framework magic and requires that or they can be safely deleted.